pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -326,11 +327,17 @@ class CreateVM extends React.Component {
                 <label className="control-label" htmlFor="vendor-select">
                     {_("OS Vendor")}
                 </label>
-                <Select.Select id="vendor-select"
-                               initial={this.state.vendor}
-                               onChange={this.onChangedValue.bind(this, 'vendor')}>
-                    {vendorSelectEntries}
-                </Select.Select>
+                <FormGroup validationState={validationStateOsVendor} bsClass='form-group ct-validation-wrapper'>
+                    <Select.Select id="vendor-select"
+                                   initial={this.state.vendor}
+                                   onChange={this.onChangedValue.bind(this, 'vendor')}>
```

Comment:
Not really necessary to re-push (as it's not new), but please let's avoid calling .bind() in component handlers. Do it once in the constructor, otherwise each change event will needlessly create a copy of the  function.

Reply:
I will repost! Thanks for catching this again :)